src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -13,6 +13,7 @@ namespace Dynamo.Search
     public class SearchDictionary<V>
     {
         private ILogger logger;
+        private static int LIMIT_SEARCH_TAG_SIZE = 300;
```

Comment:
Do we all agree that 300 characters is good enough ?
Should we increase ?

Looking through the search tags we got from Jostein, I found the following:
1. The longest search tag is 899 characters
2. There are 34 search tags (out of 6525) that are over 300 characters

Reply:
ALso this limit will be used in the Package Search window as well..(using package descriptions)

Reply:
FYI @mjkkirschner @aparajit-pratap

Reply:
This is fine by me but I'm still confused - these search tags from Jostein look like node descriptions. I'm curious if the entire description is used as a search tag, then what's the benefit of the search tags included in the `<search></search>` XML?

Reply:
@aparajit-pratap it's to inject extra search terms without adding them to the description.

So I can add `box` to the search terms for Cuboid - without needing to add `box` to the description.

Reply:
@pinzart90 what do you think about making this a hidden preference? I know it's more work, and kind of a pain, but it let's users tweak it if they run into trouble.

Reply:
@mjkkirschner I added a new assembly config for it

Reply:
oh, sorry @pinzart90 - I was not clear, by hidden preference -I just meant a preference in the dynamosettings.xml with no analog in the UI to control it. This is fine as well, users can still modify this if we want them to test something.

Reply:
@mjkkirschner if you think there is good enough reason to move to Preferences, then I can do that

Reply:
I think it would be a good idea to move it to the preference settings file so we have all preferences in one place. That is usually the one file where all such user preferences go.